vision_agent/tools/tools.py

```diff
@@ -523,8 +523,8 @@ def save_image(image: np.ndarray) -> str:
 
 def overlay_bounding_boxes(
     image: np.ndarray, bboxes: List[Dict[str, Any]]
-) -> np.ndarray:
-    """'display_bounding_boxes' is a utility function that displays bounding boxes on
+) -> Image.Image:
```

Comment:
I'm not sure about changing these to PIL images. Sometimes I ask it to visualize bounding boxes and then do something with that output, like mask or draw lines or whatever. While you can technically do this with PIL, I think it may be easier to do it with the numpy arrays/cv2. Need to test it out.

Reply:
We need PIL because the image needs to be visualized by the IPython `display()`, and `display()` doesn't understand numpy array.

Reply:
> While you can technically do this with PIL, I think it may be easier to do it with the numpy arrays/cv2. 
We don't have to do it with PIL. we could convert PIL to numpy array easily.
E.g. we could change the input of overlay_* functions to support both PIL and Numpy.

Reply:
With the upcoming video visualization feature, now i think it's better to run the `display()` function inside of every  overlay_* functions because it will constraint the vizualization behavior without adding a verbose user prompt.

Then, I don't need PIL and I can change the output back to numpy array.